app/models/installment.rb

```diff
@@ -14,7 +13,6 @@
 #  index_installments_on_franchise_id             (franchise_id)
 #  index_installments_on_media_type_and_media_id  (media_type,media_id)
 #
-# rubocop:enable Metrics/LineLength
```

Comment:
Could you undo all the places you removed `# rubocop:enable Metrics/LineLength`?